doc/source/examples/multiphysics/multiphysics.rst

```diff
@@ -2,18 +2,41 @@
 Multiphysics Examples
 *********************
 
+
+Volume of Fluid (VOF)
```

Comment:
This can be done by creating individual files for each section in the multiphysics folder (i.e., vof, heat_transfer, passive_tracer) and modifying the multiphysics.rst as follows:

```
*********************
Multiphysics Examples
*********************

.. toctree::
    :maxdepth: 1
    :glob:
    :titlesonly:

    vof
    heat_transfer
    passive_tracer

```
A more elegant option would be to create subfolders for each category, although that would require updating all the internal references accordingly.

Reply:
I'll try to do it the way you suggested without creating more sub folders.